### ISSUE TITLE
No page regions

### ISSUE THREAD

**Issue by stevelw** [stevelw/NC-News---Frontend#51]
What It Means
No page regions or ARIA landmarks were found.
Why It Matters
Regions and ARIA landmarks identify significant page areas. Most web pages should have regions defined, particularly for the main content area.
How to Fix It
If the page has visual regions or significant page areas, ensure the regions are defined with header, nav, main, footer, etc. elements.
The Algorithm... in English
No header, nav, main, footer, or aside HTML regions, or banner, navigation, main, or contentinfo landmark roles were present in the page.
Standards and Guidelines
[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)
[2.4.1 Bypass Blocks (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.1)
